meta-rz-bsp/wic/rzg2lc-sdcard.wks

```diff
@@ -0,0 +1,30 @@
+# short-description: Create SD card image for RZ/G2LC SoC
+# long-description: Creates a bootable SD card image including u-boot/atf + rootfs for Renesas RZ/G2LC SoC.
```

Comment:
Hello @hvilleneuve29 

As far as I can tell, there is nothing here tying the wics file specifically for G2LC?  
Perhaps we can make this generic for the whole `g2l-family`?

Thanks, Chris

Reply:
Yes, I will modify the description, perhaps something like:
"Create SD card image for RZ/G2L family SoC"

Reply:
... and I will rename the file.

Reply:
Thanks!